app/code/core/Mage/Tag/Model/Resource/Tag.php

```diff
@@ -109,7 +109,7 @@ protected function _afterSave(Mage_Core_Model_Abstract $object)
         $writeAdapter->insertOnDuplicate($this->getTable('tag/properties'), [
             'tag_id'            => $tagId,
             'store_id'          => $object->getStore(),
-            'base_popularity'   => (!$object->getBasePopularity()) ? 0 : $object->getBasePopularity()
+            'base_popularity'   => ($object->getBasePopularity()) ? $object->getBasePopularity() : 0
```

Comment:
There are a lot of unnecessary parentheses, but it's probably another rule.

Reply:
Yes, either other rules or php-cs-fixer. So i did not want to adjust anything manually here.